packages/modals/src/types/index.ts

```diff
@@ -101,3 +112,10 @@ export interface ITooltipModalProps
    */
   zIndex?: number;
 }
+
+export interface ITooltipModalTitleProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Updates the element's HTML tag
+   */
+  tag?: any;
```

Comment:
Do we have a mechanism for sharing types across components? If so, I'm wondering if we want to consider centralizing the `tag` interface and exporting/extending it to these components, the typography components, and any other components that may want to use this in the future.

Reply:
There is (see various forms under https://github.com/zendeskgarden/react-components/blob/main/packages/forms/src/types/index.ts), but not across packages. And when dealing with primitives, often the TypeScript juice isn't worth the squeeze.